### DISCUSSION TITLE
Please let me know some of the big customers who use this service

### DISCUSSION THREAD

**Discussion by vahidalvandi** [docker-mailserver/docker-mailserver#3460] [General]
Because some companies such as **redacted** offer the same service and how can we be sure that this service is safe and efficient?

**Comment by fbartels** [comment]
Hi @vahidalvandi,

I have redacted the name of the third company in your post to avoid people seeing this as some kind of endorsement.

But essentially what you linked to is a commercial company selling a software to run a mailserver with additional features such as document editing. Docker-mailserver however is a free to use open source project, there are users, but not customers that are known to someone dealing with invoices. The only real metrics are stuff like pulls of the container, stars here on GitHub, and engagement of users in issues, pull requests and discussion.

This my personal opinion on this at least.

**Comment by vahidalvandi** [comment]
@fbartels 
Thank you for your answer, I have a question about the dependencies of the container. Sub-softwares like postfix are updated in images that are updated. Of course, if docker mail server is not up to date.

I mean, should the updated docker mail server have postfix in its image or is it updated independently and has nothing to do with the docker mail server image.

**Reply by fbartels** [reply]
Postfix is part of the container and will therefore also be updated, when you pull in a new release.
